tests/static/locators.html

```diff
@@ -30,6 +30,9 @@
         document.querySelector('#inputCheckbox').addEventListener(
             'change', e => { window.check = e.currentTarget.checked; }, false
         );
+        document.querySelector('#inputText').addEventListener(
+            'mousemove', e => { result = true; }, false
```

Comment:
```suggestion
            'mousemove', e => { window.result = true; }, false
```

Same for the `click` handler.

Reply:
Thanks, see https://github.com/grafana/xk6-browser/pull/400/commits/c3a90bd8f2ebad00c01ffcec29718e30711461d6.